esp-hal-common/src/systimer.rs

```diff
@@ -202,14 +202,14 @@ impl<const CHANNEL: u8> Alarm<Target, CHANNEL> {
 
 impl<const CHANNEL: u8> Alarm<Periodic, CHANNEL> {
     pub fn set_period(&self, period: fugit::HertzU32) {
-        let time_period: MillisDurationU32 = period.into_duration();
+        let time_period: MicrosDurationU32 = period.into_duration();
         let cycles = time_period.ticks();
         self.configure(|tconf, hi, lo| unsafe {
             tconf.write(|w| {
                 w.target0_period_mode()
                     .set_bit()
                     .target0_period()
-                    .bits(cycles * (SystemTimer::TICKS_PER_SECOND as u32 / 1000))
+                    .bits(cycles * (SystemTimer::TICKS_PER_SECOND as u32 / 1000_000))
```

Comment:
If this is actually correct (after a few minutes of trying to understand fugit, I think it is), would you mind renaming `cycles` to `us` or `micros` or something similar? This is very confusing (to me) currently.

Reply:
Good call, I've changed the local name to `us`.